tests/src/Unit/OgAccessHookTest.php

```diff
@@ -22,25 +21,38 @@ public function setUp() {
   }
 
   /**
+   * Tests that an entity which is not a group or group content is ignored.
+   *
    * @dataProvider operationProvider
    */
   public function testNotContentEntity($operation) {
     $entity = $this->prophesize(EntityInterface::class);
     $access = og_entity_access($entity->reveal(), $operation, $this->user->reveal());
+
+    // An entity which is not a group or group content should always return
+    // neutral, since we have no opinion over it.
     $this->assertTrue($access->isNeutral());
   }
 
   /**
+   * Tests that an administrator with 'administer group' permission has access.
+   *
    * @dataProvider operationProvider
    */
   public function testGetEntityGroups($operation) {
     $this->user->hasPermission(OgAccess::ADMINISTER_GROUP_PERMISSION)->willReturn(TRUE);
-    $user_entity_access = og_entity_access($this->entity->reveal(), $operation, $this->user->reveal());
+    $user_entity_access = og_entity_access($this->groupContentEntity->reveal(), $operation, $this->user->reveal());
+
+    // @todo This is strange, 'view' is not part of the operations supplied by
+    //   ::operationProvider(). And why would a group administrator be allowed
+    //   access to all operations, except 'view'? Shouldn't this also return
+    //   'allowed'?
```

Comment:
This is puzzling to me. It seems to be the intention to hand off the 'view' permission to core, but this is counterintuitive. What would be the use case of not granting 'view' access to a group admin, while all the other permissions and operations are granted?